packages/ibm-products/src/components/Guidebanner/Guidebanner.js

```diff
@@ -69,8 +71,9 @@ export let Guidebanner = React.forwardRef(
         className={cx(
           blockClass,
           className,
-          [collapsible ? `${blockClass}__collapsible` : null],
-          [isCollapsed ? `${blockClass}__collapsible-collapsed` : null]
+          collapsible && `${blockClass}__collapsible`,
+          isCollapsed && `${blockClass}__collapsible-collapsed`,
```

Comment:
Extra: simplified code.

Reply:
Haha, is this pulling from your recent convo with @davidmenendez 😆

Reply:
Actually, I had completely missed his comment while making these changes. :)